docs/user_guide/hopsworks/python.rst

```diff
@@ -8,11 +8,11 @@ Python Environment Basics
 
 Hopsworks provides a premade environment for machine learning and data science development using python 3.7. The environment contain the most popular machine learning libraries including TensorFlow, Keras, PyTorch and scikit-learn.
 
-The environment ensures compatibility between the machine learning libraries and CUDA/cuDNN in case of NVIDIA GPUs and ROCm in case of AMD GPUs.
+The environment ensures compatibility between the CUDA version and the installed TensorFlow and PyTorch versions for applications using NVIDIA GPUs.
 
-In each project there is an Anaconda environment, which is replicated at all data processing hosts in the cluster. This means that whenever an operation is issued through the UI such as, installing or uninstalling a library, it is replicated on all the Anaconda environments which keeps them in sync.
+For each project, an Anaconda environment can be managed, which is is stored in a Docker image. Issuing an operation from the Python UI such as installing or uninstalling a library also updates the Anaconda environment.
 
-Each python application running in the project, such as for example PySpark, uses the project's environment.
+Each python application running in the project, such as for example PySpark, uses the project's Anaconda environment. The Docker image is pulled on all hosts to ensure that the running applications gets the updated environment as libraries gets added and are removed.
```

Comment:
The Docker image is pulled on all hosts to ensure that the running application gets the latest version of the Anaconda environment, as libraries can be installed and uninstalled at any time in the project.